### DISCUSSION TITLE
Thought it meant "daseru"

### DISCUSSION THREAD

**Discussion by RickCogley** [TomWright/dasel#121] [General]
Thanks for `dasel`. What a wonderful tool.  

I know `dasel` is named from "data selector", but when I first saw it I thought it was named with the Japanese "daseru" (出せる) in mind, which means "can give / output". Not important of course but just thought I'd mention.

**Comment by TomWright** [comment]
That's an interesting coincidence. I had no idea but it does fit nicely!

**Comment by groteck** [comment]
Maybe for a ui Like [jqp](https://github.com/noahgorstein/jqp)

**Reply by TomWright** [reply]
If this is a feature suggestion please raise another discussion under the ideas category.
